### PR TITLE
Add queries using JOIN and Reference

### PR DESCRIPTION
**Pull request by devalibello**
This pull request adds more queries using JOIN and Foreign Key Reference. It also adds more tables to the database. The various queries that were performed are provided in screenshots below.

    - What animals belong to Melody Pond?

    - List of all animals that are pokemon (their type is Pokemon).

    - List all owners and their animals, remember to include those that don't own any animal.

    - How many animals are there per species?

    - List all Digimon owned by Jennifer Orwell.

    - List all animals owned by Dean Winchester that haven't tried to escape.

    - Who owns the most animals?

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -5,5 +5,52 @@ CREATE TABLE animals (
...
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(300)
+);
+
+ALTER TABLE animals
+ADD CONSTRAINT fk_species
+FOREIGN KEY (species_id) REFERENCES species(id);
+
+ALTER TABLE animals
```
Comment (Shedrack-Sunday) [inline schema.sql]: - [ ] Kindly add  a modification to  remove the species column entirely from the animal's table

### Instructions:
image

File: data.sql [hunk 2]
```diff
@@ -3,75 +3,104 @@ INSERT INTO animals (
...
+    ),
+    (
+        'Blossom',
+        '1998-10-13',
+        3,
+        TRUE,
+        17,
+        1,
+        2
     );
```
Comment (Shedrack-Sunday) [inline data.sql]: - [ ] You're missing some major insertions in the Data.sql file. Below you'll find a screenshot of them all.

### Instructions:
image

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @devalibello      👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the SQL queries✔️
   
# Required Changes  🌵  

- [ ] You've done well, However, kindly check some key point note I pointed out to meet the requirements before moving to the next project 👍 
- [ ] Please add a License file to the root of your README File, and delete the guide instructions 
- [ ] Kindly snap the screenshots after making the changes below 


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: schema.sql (hunk 1 above)
Comment (devalibello) [reply schema.sql]: Hello @Shedrack-Sunday
The section of the code you highlighted adds a foreign key reference to the species_id column. The species column has already been removed from the animals table as indicated in the screenshot below.

image
I have added a comment to the section of the code you highlighted for clarity.

Thanks 😄

**Comment by devalibello** [comment]
Hello,

Here are the screenshots of all the required changes (insertions to data.sql).
Thanks 😄 

image
image
image

**Review by prolajumokeoni — APPROVED** [review]
Hi @devalibello 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented ✅ 
 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**